README.md

```diff
@@ -84,7 +84,7 @@ kubectl delete --ignore-not-found=true -f manifests/ -f manifests/setup
 To try out this stack, start [minikube](https://github.com/kubernetes/minikube) with the following command:
 
 ```shell
-$ minikube delete && minikube start --kubernetes-version=v1.23.0 --memory=6g --bootstrapper=kubeadm --extra-config=kubelet.authentication-token-webhook=true --extra-config=kubelet.authorization-mode=Webhook --extra-config=scheduler.bind-address=0.0.0.0 --extra-config=controller-manager.bind-address=0.0.0.0
+$ minikube delete && minikube start --container-runtime=containerd --kubernetes-version=v1.31.0 --memory=8g --bootstrapper=kubeadm --extra-config=kubelet.authentication-token-webhook=true --extra-config=kubelet.authorization-mode=Webhook --extra-config=scheduler.bind-address=0.0.0.0 --extra-config=controller-manager.bind-address=0.0.0.0
```

Comment:
Lets update to latest one

```suggestion
$ minikube delete && minikube start --container-runtime=containerd --kubernetes-version=v1.32.3 --memory=8g --bootstrapper=kubeadm --extra-config=kubelet.authentication-token-webhook=true --extra-config=kubelet.authorization-mode=Webhook --extra-config=scheduler.bind-address=0.0.0.0 --extra-config=controller-manager.bind-address=0.0.0.0
```

any reason updating memory to 8g?

Reply:
> any reason updating memory to 8g?

i needed more memory for one particular scenario, we can leave 6g if you want